vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -291,7 +287,6 @@ components:
           type: string
         iommu:
           type: boolean
-          default: false
```

Comment:
That is still valid, the iommu default will be set to `false`.

Reply:
Yes my bad, I didn't realize this was simply for documentation purpose.
I need to add the default value for other fields too.

Reply:
Done!